prboom2/INSTALL

```diff
@@ -74,6 +51,9 @@ Basic Installation
      If you have a system with many users, you should read the license for
      your version of Doom, and make sure only those users allowed to use it
      can access the IWAD file.
+     
+     You can also aquire FreeDoom from https://freedoom.github.io/ which is a 
+     free replacement to the iD Doom iWADs (although if you intend to upload demos to DSDA this iWAD isn't sufficient) 
```

Comment:
I think you can remove the `(although if you intend to upload demos to DSDA this iWAD isn't sufficient)` from here. That's certainly correct but it's probably not in the scope of the INSTALL doc :^)

Reply:
Sorted that, sorry. I forgot I wasn't committing this to the DSDA branch.